integration/todo/components.qwik.tsx

```diff
@@ -19,7 +19,7 @@ import {
   Todos,
   toggleItem,
   updateFilter,
-} from './state';
+} from './state.qwik';
```

Comment:
What is the thinking behind renaming `state` to `state.qwik`?

Reply:
It should not be required, i just renamed all files. But since it does not contain any qwik it should not be necesary